cmd/validator/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"os"
+
+	"github.com/operator-framework/operator-lifecycle-manager/cmd/validator/schema"
+)
+
+func main() {
+	// TODO(alecmerdler): Get manifest directory from args
+	manifestDir := os.Args[1]
+
+	// FIXME(alecmerdler): `TestCatalogVersions` is meant to run against built catalog configmaps
+	// schema.TestCatalogVersions(manifestDir)
```

Comment:
We can simplify this check to simply: `for all manifests with replaces field: check that csv being replaced actually exists`. That should make it easier to implement (and generalize for operator-manifests)